tools/tpm-tests-on-qemu.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Some of the EVE unit-tests require a TPM to run, on Github runners there is
+# no TPM, and as a result, we skip those tests. It is possible to simulate a TPM
+# on the host using the tpm_vtpm_proxy kernel module and swtp, but there is a good 
```

Comment:
A yetus/blanks issue to fix on this line.

Reply:
fixed.